modules/custom/ymca_sync/src/Form/SettingsForm.php

```diff
@@ -38,12 +38,12 @@ public function getFormId() {
   /**
    * SettingsForm constructor.
    *
-   * @param ConfigFactoryInterface $config_factory
-   *   Config Factory.
-   * @param SyncRepository $syncers
-   *   Sync Repo.
-   * @param ModuleHandlerInterface $module_handler
-   *   Sync Repo.
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   Parameter \Drupal\Core\Config\ConfigFactoryInterface.
+   * @param \Drupal\ymca_sync\SyncRepository $syncers
+   *   Parameter \Drupal\ymca_sync\SyncRepository.
+   * @param \Drupal\Core\Extension\ModuleHandlerInterface $module_handler
+   *   Parameter \Drupal\Core\Extension\ModuleHandlerInterface.
```

Comment:
I believe we should use standard Drupal approach for these comments.
"The config factory service."
"The sync repository."
"The module handler to invoke the alter hook with." || "The module handler."